en/02_Developer_Guides/01_Templates/01_Syntax.md

```diff
@@ -271,7 +271,7 @@ include.
 
 ## Looping over lists
 
-The `<% loop %>` tag is used to iterate or loop over a collection of items such as [DataList](api:SilverStripe\ORM\DataList) or an [ArrayList](api:SilverStripe\ORM\ArrayList)
+The `<% loop %>` tag is used to iterate or loop over a collection of items such as a native PHP array or `Iterator` class, [DataList](api:SilverStripe\ORM\DataList), or an [ArrayList](api:SilverStripe\ORM\ArrayList)
```

Comment:
Leaving this not done for now pending a final decision on the main PR - but if it's merged as-is (i.e. wrapping in `ArrayList`) I'll make this change.
```suggestion
The `<% loop %>` tag is used to iterate or loop over a collection of items such as a native PHP array, [DataList](api:SilverStripe\ORM\DataList), or an [ArrayList](api:SilverStripe\ORM\ArrayList)
```